std/experimental/ndslice/iteration.d

```diff
@@ -104,6 +104,8 @@ import std.meta;
 import std.experimental.ndslice.internal;
 import std.experimental.ndslice.slice; //: Slice;
 
+@trusted:
```

Comment:
Having `@trusted` at the top of the file is a very bad idea. It runs a high risk of affecting things that you didn't intend to be `@trusted`.